src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -146,7 +147,9 @@ class CodeLongevity(private val serverRepo: Repo,
                     git: Git) {
     val repo: Repository = git.repository
     val head: RevCommit =
-        RevWalk(repo).parseCommit(repo.resolve(RepoHelper.MASTER_BRANCH))
+        try { RevWalk(repo).parseCommit(repo.resolve("HEAD")) }
```

Comment:
All hashers should be consistent in what branch they hash. We have agreement of hashing master branch, ref defined in RepoHelper.MASTER_BRANCH.